crates/proto/src/xfer/dns_request.rs

```diff
@@ -103,3 +133,9 @@ impl From<Message> for DnsRequest {
         Self::new(message, DnsRequestOptions::default())
     }
 }
+
+// TODO: this should be configurable
+// > An EDNS buffer size of 1232 bytes will avoid fragmentation on nearly all current networks.
+// https://dnsflagday.net/2020/
+#[cfg(feature = "std")]
+const MAX_PAYLOAD_LEN: u16 = 1232;
```

Comment:
Unrelated to the move, I wonder if this should be used in `Edns::default()` instead of the current 512 value?